microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -79,126 +79,148 @@ In **Guest** mode, autofill is not available and new autofill entries aren't add
-| _page information_ | The webpage content, or the content of the document, that's open in the active tab of the Microsoft Edge browser. |
+You can access Collections from an InPrivate window when using a signed-out profile.
 
 
 <!-- ====================================================================== -->
-## Cast
-
-Cast in Microsoft Edge lets you display your media to another screen using Google Cast.  To access Cast, open **Settings and more (...)** > **More tools** > **Cast media to device**.  Cast relies on the Media Router extension not automatically installed with Microsoft Edge.  When you first use Cast, Microsoft Edge prompts for permission to install the Media Router extension.
+## Compose
 
-Select **restart** to install the Media Router extensions from the Chrome Web Store. On startup and at regular intervals, Microsoft Edge sends update requests to the Chrome Web Store. Update requests include basic data about your version of Microsoft Edge. Regular update requests keep the Media Router extension up to date.
+The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.
 
-Google may collect some data associated with the Media Router extension. To uninstall the Media Router extension, go to `edge://flags#edge-on-demand-media-router` and change the setting. Uninstalling also stops updates from the Chrome Web Store. The extension is hidden and does not appear on the **Installed extensions** list. For the **Installed extensions** list, go to `edge://extensions`.
 
+<!-- ------------------------------ -->
+#### Use Compose to draft text within the Edge Sidebar
 
-<!-- ====================================================================== -->
-## Collections
+In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections.
 
-The Collections feature in Microsoft Edge allows you to save web pages, text, images, and videos from the web. You can organize the content based on the projects, events, and interests that matter to you. You can also save notes in your collections and add notes to individual items as well. Collections uses the Microsoft Edge Entity Extraction service to enrich the content that you collect by including thumbnails and metadata, such as price and star rating. For more information, see [Entity extraction](#entity-extraction) below.
+Text that's generated by Compose can be copied, or inserted directly into the webpage.
 
-The Collections feature is not available in **Guest** mode or Kids Mode.
+To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button, and then select **Compose**.  Or, press **Ctrl+Shift+.**.  Text that you select or enter is sent to an internal Microsoft service that runs a Large Language Model (LLM).
```

Comment:
```suggestion
To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button, and then select **Compose**.  Or, press **Ctrl+Shift+.**. 
```
This is already noted in the 'Privacy and data protections for Compose' section. Removing for clarity and conciseness.

Reply:
Removed sentence, although that later section doesn't have that exact sentence.  has paragraph:
```
Text that you've highlighted or inputted for Compose 
is sent to Microsoft after you click the **Rewrite** or 
**Generate** button, not while you're selecting text.  Each 
request contains the input prompt text, formatting selections, 
and IDs that are associated with your Microsoft Edge profile 
and device, to reference for subsequent requests.  Requests 
for Compose are sent over a secure HTTPS connection.
```
so, after new commit, whitepaper won't contain "large language" at all.
@apryzgoda @majenny